src/views/Drive.vue

```diff
@@ -2375,8 +2375,8 @@ module.exports = {
                 return peergos.shared.user.JavaScriptPoster.emptyArray();
         	} else {
                 var canvas = document.createElement('canvas');
-                canvas.width = window.innerWidth - 100;
-                canvas.height = window.innerHeight - 100;
+                canvas.width = 800;
```

Comment:
what are these numbers? how does this look on mobile?

Reply:
previously a blank image was created using the size of the screen on whatever device was being used. Using a standard size seems more apporiate. Image editor can resize if desired.